ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/XARecoveryModuleUnitTest.java

```diff
@@ -441,7 +446,6 @@ public String getProductVersion() {
                 public String getJndiName() {
                     return "test";
                 }
-
                 int count = 0;
```

Comment:
Thanks for changing it back to count. Quite a few comments still refer to `recoverCount`, will you change those to count too please.

Reply:
Sure, I forgot about that. Updated, thanks